### PR TITLE
Modified the play not found message with search term

### PR DESCRIPTION
**Pull request by vasantisuthar**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/common/playlists/PlayList.jsx [hunk 1]
```diff
@@ -23,7 +26,7 @@ const PlayList = () => {
     return (
       <div className='play-not-found'>
         <ImageOops className='play-not-found-image' />
-        <p className='page-404-lead'>Play not found</p>
+        <p className='page-404-lead'>Play not found for "{ searchTerm }" </p>
```
Comment (Sachin-chaurasiya) [inline src/common/playlists/PlayList.jsx:29]: @vasantisuthar , Can we add check here that show `for " searchTerm" ` only if searchTerm is not a falsy value?

**Review by Sachin-chaurasiya — COMMENTED** [review]
Nit:

**Comment by atapas** [comment]
@Sachin-chaurasiya could you please review this one?

File: src/common/playlists/PlayList.jsx (hunk 1 above)
Comment (Sachin-chaurasiya) [reply src/common/playlists/PlayList.jsx:30]: Hey @vasantisuthar, Can you address the review comment?

File: src/common/playlists/PlayList.jsx [hunk 2]
```diff
@@ -24,7 +27,7 @@ const PlayList = () => {
...
```
Comment (atapas) [inline src/common/playlists/PlayList.jsx:30]: @vasantisuthar You also need to handle for the filter criteria. For filter I would suggest the text to be "matched filter criteria" than the selected values.

**Review by atapas — CHANGES_REQUESTED** [review]

File: src/common/playlists/PlayList.jsx (hunk 1 above)
Comment (vasantisuthar) [reply src/common/playlists/PlayList.jsx:30]: > @vasantisuthar , Can we add check here that show `for "searchTerm" ` only if searchTerm is not a falsy value?

I didn't get this, can you please elaborate?

Comment (Sachin-chaurasiya) [reply src/common/playlists/PlayList.jsx:30]: Sure, @vasantisuthar , we need to handle the case when `searchTerm` is a false value like empty, undefined, etc.
So if `searchTerm` is a falsy value then we should just show `Play not found`.

Comment (vasantisuthar) [reply src/common/playlists/PlayList.jsx:30]: okay

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Readable

File: src/common/playlists/PlayList.jsx [hunk 3]
```diff
@@ -23,8 +32,12 @@ const PlayList = () => {
   if (plays?.length === 0) {
     return (
       <div className='play-not-found'>
-        <ImageOops className='play-not-found-image' />
-        <p className='page-404-lead'>Play not found</p>
+        <ImageOops className='play-not-found-image' />  
+        <p className='page-404-lead'>Play not found 
+        {hasFilterQuery? " for matched filter criteria" : null}
+        {hasFilterQuery && searchTerm ? " and ": null}
+        {searchTerm ? " for " + searchTerm : null}
```
Comment (Sachin-chaurasiya) [inline src/common/playlists/PlayList.jsx:39]:
```suggestion
        {searchTerm ? ` for  ${searchTerm}` : null}
```

File: src/common/playlists/PlayList.jsx [hunk 4]
```diff
@@ -23,8 +32,12 @@ const PlayList = () => {
   if (plays?.length === 0) {
     return (
       <div className='play-not-found'>
-        <ImageOops className='play-not-found-image' />
-        <p className='page-404-lead'>Play not found</p>
+        <ImageOops className='play-not-found-image' />  
+        <p className='page-404-lead'>Play not found 
+        {hasFilterQuery? " for matched filter criteria" : null}
+        {hasFilterQuery && searchTerm ? " and ": null}
```
Comment (Sachin-chaurasiya) [inline src/common/playlists/PlayList.jsx:38]:
```suggestion
        {hasFilterQuery && searchTerm ? " and": null}
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]
Nit:

**Comment by atapas** [comment]
@vasantisuthar what's pending on this?

**Review by Sachin-chaurasiya — DISMISSED** [review]

File: src/common/playlists/PlayList.jsx [hunk 5]
```diff
@@ -23,8 +32,12 @@ const PlayList = () => {
   if (plays?.length === 0) {
     return (
       <div className='play-not-found'>
-        <ImageOops className='play-not-found-image' />
-        <p className='page-404-lead'>Play not found</p>
+        <ImageOops className='play-not-found-image' />  
+        <p className='page-404-lead'>Play not found 
+        {hasFilterQuery? " for matched filter criteria" : null}
+        {hasFilterQuery && searchTerm ? " and": null}
+        {searchTerm ? ` for  ${searchTerm}` : null}
```
Comment (koustov) [inline src/common/playlists/PlayList.jsx:39]: A tiny observation,
This is a little complex from the readability front. Can you simplify it? Something like creating a method for doing this job in a more readable way?

**Review by koustov — CHANGES_REQUESTED** [review]
A very tiny comment

**Comment by atapas** [comment]
@vasantisuthar What's pending in this?

**Comment by atapas** [comment]
Closing this PR as No Response for close to a couple of months. Please request to open if still working on it.

**Pull request closed without merging**